compiler_opt/rl/compilation_runner_test.py

```diff
@@ -28,6 +28,8 @@
 
 from compiler_opt.rl import compilation_runner
 from compiler_opt.rl import constant
+from compiler_opt.rl.adt import ModuleSpec
+from compiler_opt.rl.problem_configuration import get_command_line_for_bundle
```

Comment:
the test for get_command_line_for_bundle related should be in the *_test.py file that * == the module the function belongs to